overlord/devicestate/firstboot_test.go

```diff
@@ -58,27 +58,28 @@ import (
 )
 
 type FirstBootTestSuite struct {
-	systemctl *testutil.MockCmd
+	testutil.BaseTest
 
-	storeSigning *assertstest.StoreStack
-	restore      func()
+	systemctl *testutil.MockCmd
 
-	brands *assertstest.SigningAccounts
+	seedtest.TestingSeed
```

Comment:
Same question as above, maybe a small comment about the significance of this one?